dame_flame/post_matching.py

```diff
@@ -0,0 +1,167 @@
+#%% Imports
+import pandas as pd
+import numpy as np
+import dame_flame
+import time
+from . import DAME_FLAME
+
+#%% Original algorithm speed test
+df = pd.read_csv("data/data.csv")
+
+start_original = time.time()
+result = dame_flame.DAME_FLAME.FLAME(input_data=df, holdout_data = df,verbose=0,treatment_column_name = 'treated', outcome_column_name = 'outcome',repeats = True)
+end_original = time.time()
+print(end_original - start_original)
+
+#%% Modified algorithm speed test
+df = pd.read_csv("data/data.csv")
+
+start_new = time.time()
+result_new = DAME_FLAME.FLAME(input_data=df, holdout_data = df,verbose=0,treatment_column_name = 'treated', outcome_column_name = 'outcome',repeats = True)
+end_new = time.time()
+print(end_new - start_new)
+
+#%% MG function
+def MG(return_df, unit_ids, input_data):
+    '''
+    This function returns the main matched groups for all specified unit
+    indices
+
+    Args:
+        return_df : output of DAME or FLAME
```

Comment:
Since you updated what this looks like, could you update the documentation that describes this? I imagine its definitely described in the README but possibly also in DAME_FLAME.py